src/EdFi.AnalyticsMiddleTier.DataStandard32/Base/PostgreSQL/0025-View-StudentSchoolDim-Alter.sql

```diff
@@ -39,7 +38,8 @@ SELECT
                 ,(schoolEdOrg.LastModifiedDate)
                 ,(districtEdOrg.LastModifiedDate)
             ) AS VALUE(MaxLastModifiedDate)
-        ) AS LastModifiedDate
+        ) AS LastModifiedDate,
+		Student.BirthDate
```

Comment:
PostgreSQL does not like to have new columns show up in the middle of the list - before this change, it assumed that `EnrollmentDateKey` was being _replaced_ by `BirthDate`. So the simple solution was to move it to the end of the list.